README.md

```diff
@@ -154,8 +154,9 @@ Once all the services are green, open the gateway service: http://localhost:8080
 You can delete the Azure resource group, which includes the database.
 
 ```bash
-az group delete --name australia-east --no-wait --yes
+az group delete --name australia-east
 ```
+This will take several minutes to finish. Once it is done, it is a good idea to log into the [Azure Portal](https://portal.azure.com) and make sure all of the resouces have been deleted: the resoure group, the Cosmos DB instance, and the AKS group.
```

Comment:
```suggestion
This will take several minutes to finish. Once it is done, it is a good idea to log into the [Azure Portal](https://portal.azure.com) and make sure all of the resources have been deleted: the resource group, the Cosmos DB instance, and the AKS group.
```